geocoder-and-reverse-geocoder-android/app/src/main/AndroidManifest.xml

```diff
@@ -36,12 +36,12 @@
         <service
             android:name="com.here.android.mpa.service.MapService"
             android:label="HereMapService"
-            android:process="global.Here.Map.Service.v2"
-            android:exported="true">
+            android:process="global.Here.Map.Service.v2">
             <intent-filter>
-                <action android:name="com.here.android.mpa.service.MapService"></action>
+                <!-- When using MapSettings.setIsolatedDiskCacheRootPath(...) make sure to use this value: -->
+                <action android:name="com.here.android.example.geocoding.MapService" />
```

Comment:
What happens if we don't use an isolated disk cache but still change the service name?
Isn't the MapServiceClient now expecting the old name?

In the example I linked in the other comment I see that it's either:
- keep everything with the default values
- replace the service name and set the isolated disk cache using the new service name

This case is something in between